### PR TITLE
[10.0] mig account_tax_balance sbi

### PR DESCRIPTION
**Pull request by sbidoul**

### CODE REVIEW

**Comment by eLBati** [comment]
@sbidoul thanks!

Could you check the test error
```
2016-11-19 13:26:18,704 5765 ERROR openerp_test odoo.addons.account_tax_balance.tests.test_account_tax_balance: `   File "/home/travis/build/OCA/account-financial-reporting/account_tax_balance/tests/test_account_tax_balance.py", line 70, in test_tax_balance

2016-11-19 13:26:18,704 5765 ERROR openerp_test odoo.addons.account_tax_balance.tests.test_account_tax_balance: `     self.assertEquals(tax.base_balance, 100.)

2016-11-19 13:26:18,704 5765 ERROR openerp_test odoo.addons.account_tax_balance.tests.test_account_tax_balance: ` AssertionError: 0.0 != 100.0
```
?

**Comment by eLBati** [comment]
Also, rebasing the branch on top of the current 10.0 should fix the runbot issue

**Comment by gurneyalex** [comment]
@sbidoul rebasing on top of 10.0 should solve the runbot skipping the build.

**Comment by sbidoul** [comment]
@eLBati rebased. I don't have time to dig into the failed test right now, though.

**Comment by eLBati** [comment]
@sbidoul fixed with https://github.com/OCA/account-financial-reporting/pull/250/commits/b1eb23477f4d7ba320ab93302576032b6b257a87

**Review by eLBati — APPROVED** [review]

File: account_tax_balance/README.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+Accounting --> Reporting --> Taxes Balance
+
+Select the company, the date range, the target moves and 'open taxes'
+
+.. figure:: /account_tax_balance/static/description/tax_balance.png
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/91/9.0
```
Comment (pedrobaeza) [inline account_tax_balance/README.rst]: Change version

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
The menu should have another level (with an extra parent), although it seems weird, but if not, with the web_responsive module is mixed between other menus. Can you please do it?

**Comment by eLBati** [comment]
@pedrobaeza thanks, done

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**